### ISSUE TITLE
Bilder und Texte von neuen Zeugnissen

### ISSUE THREAD

**Issue by gerritbruening** [faustedition/xml#12]
Tabellenspalte "Status" (erste Spalte) sortieren, Einträge "NEU".

**Comment by gerritbruening** [comment]
Es fehlt auch `lfd. Nr. (neuestens)` Nr. `377`.

Zu benutzende Edition: Fehling.

**Comment by gerritbruening** [comment]
Neu ist voraussichtlich auch https://github.com/faustedition/xml/issues/14.

**Comment by gerritbruening** [comment]
Ersetzt werden muss auch burkhardt1870_0009, burkhardt1870_0010, burkhardt1870_0011 durch ebd., S. 8f. (richtig ist das vorhergehende Zeugnis)

**Comment by gerritbruening** [comment]
Es fehlt auch Dingelstedt. Bilder sind vorhanden: P:\digitalisate\Entstehungszeugnisse\Digitalisate erste Lieferung veraltet\Entstehungszeugnisse Variante 1 hoechstmoegliche Bildqualitaet\dingelstedt1863

**Comment by gerritbruening** [comment]
Von https://github.com/faustedition/xml/issues/8 sind ein paar Fehler übrig, die aus der ersten Abteilung der WA stammen (siehe https://faustedition.uni-wuerzburg.de/wiki/index.php/Zeno.org-WA_Fehlerliste).
Haben wir von diesen Zeugnissen elektronische Texte? Oder gehören diese auch zu den neuen?

**Comment by gerritbruening** [comment]
Diese und verwandte Issues werden in folgender Tabelle verwaltet:

https://docs.google.com/spreadsheets/d/11Z043ZghUsXGrOiWsEJsopcub-SdeYX92rPYkfcpfiE/edit?usp=sharing

**Comment by gerritbruening** [comment]
Wahrscheinlich wäre es am besten, alle "Nachzügler" in einer Tabelle zu sammeln, deren Digitalisierung also nicht verstreut in der großen Excel-Tabelle zu verwalten.

**Comment by sandrakrause** [comment]
Neue, erweiterte Tabelle zur Verwaltung fehlender Zeugnisse ist jetzt hier zu finden:
https://docs.google.com/spreadsheets/d/17SS6Z5kGzqwiX0T7YIx8AEQ9x99KyLaTKK1OkOY6pq4/edit?usp=sharing

**Comment by sandrakrause** [comment]
Elemente, die durch ihren Inhalt ersetzt werden können:
<em></em>
<font></font>
<strong></strong>

**Comment by gerritbruening** [comment]
Siehe zuletzt https://faustedition.uni-wuerzburg.de/wiki/index.php/10._Juli_2017#Entstehungszeugnisse_2:
* DP soll die Wahl der Vorlagen prüfen
* SK soll die noch zu digitalisierenden Zeugnisse in Buchform beschaffen,
* schwierige Bücher vorab identifizieren und mit Fr. Woldemariam klären, ob diese einen Teil der Digitalisierung übernehmen oder SK darin einweisen kann

**Comment by gerritbruening** [comment]
> DP soll die Wahl der Vorlagen prüfen

(https://faustedition.uni-wuerzburg.de/wiki/index.php/10._Juli_2017#Entstehungszeugnisse_2)

**Comment by sandrakrause** [comment]
Probeseiten der verschiedenen zu digitalisierenden Dokumente sind abgelegt unter:
Z:(faust)\digitalisate\Entstehungszeugnisse\nachdigitalisierung\- Probeseiten\Probeseiten.pdf

Es handelt sich aktuell um insgesamt 168 zu digitalisierende Seiten.
Bei angenommenen 3000 Zeichen pro Seite ergibt das hochgerechnet insgesamt 504.000 Zeichen.
(Diese Daten sind auch in der Übersichtstabelle auf dem zweiten Tabellenblatt aufgelistet)

**Comment by thvitt** [comment]
Word mit Formatvorlagen verwenden, vor allem für sowas wie Vers vs. Absatz, Bylines in Briefen, Überschriften etc.; halt möglichst alles was im XML irgendwie ausgezeichnet werden soll. Vorgabe keine anderen Schriften für Sonderzeichen, sondern Unicode-Sonderzeichen verwenden (Windows-Zeichentabelle?). 

Dann docxtotei (TEI-Stylesheets / OxGarage) + ein paar einfache Regeln, um aus sowas wie `<p rend="Vers">` ein `<l>` zu machen. An den Problemfällen den Studierenden visuelles vs semantisches Markup und die Probleme von wordbasierten Workflows erläutern :-)

**Comment by gerritbruening** [comment]
Danke für deine Rückmeldung! Das Problem ist, wenn ich die Regeln für die Zuweisung der Formatvorlagen vorgeben könnte, könnte ich auch gleich das XML vorgeben. Die Zeit für das herausholen von semantischen Markup aus typografischem Erscheinungsbild mir aber gerade.
Das mit den Sonderzeichen gebe ich schon mal weiter.

**Comment by thvitt** [comment]
achso.

naja, dann bleiben ja nicht viele Alternativen. Letztlich heißt das doch, diese intellektuelle Leistung nach hinten zu verlagern und dann anhand der abgetippten Daten gucken, ob man da Regeln formulieren kann oder automatisch nachauszeichnet. Dann kann man auch über dne Workflow entscheiden,w enn die daten da sind.

**Comment by gerritbruening** [comment]
Genau, die intellektuelle Leistung wird nach hinten verlagert. Die Studierenden sollen ja ruhig händisch auszeichnen.
Einzige Unbekannte für mich ist das Verhalten der Stylesheets bzw der oxgarage. Also ob da alle Formatierungen wie zentriert usw. rüberkommen.

**Comment by thvitt** [comment]
Soweit ich weiß ja. Man kann das auch beeinflussen. Man kann sich ja ggf. auch die Option freihalten, die DOCX nachzubearbeiten – z.B. Überschriften einführen – um dann das entsprechende Grouping durch die Stylesheets durchführen zu lassen.

**Comment by gerritbruening** [comment]
> Man kann sich ja ggf. auch die Option freihalten, die DOCX nachzubearbeiten – z.B. Überschriften einführen

Das ist eine gute Idee, machen wir.
Sehen wir weiter, wenn die Daten da sind und die Studierenden in der Spur.

**Comment by dietmarpravida** [comment]
was ist daraus denn geworden?

**Comment by sandrakrause** [comment]
Zu den letzten Überlegungen kann ich nichts sagen.
Prinzipiell liegen die Texte der auffindbaren Zeugnisse vor, sind aber noch nicht alle codiert.
@gerritbruening Wie hoch steht das auf der Prioritätenliste? (D.h. müssen die für die 1.0 alle fertig werden?)
Und wie steht es mit den Zeugnissen, die Studierenden zugewiesen wurden? Haben die geliefert oder muss ich da noch welche übernehmen?

**Comment by gerritbruening** [comment]
Sie könnten ja schauen, ob Sie die Zeugnisse mit dem im Seminar praktizierten Workflow basal aufbereitet kriegen, plus lassen https://github.com/faustedition/faust-gen-html/blob/master/tools/docx-cleanup.xsl durchlaufen.

**Comment by sandrakrause** [comment]
Das habe ich für (fast) alle bereits vorbereitet. Trotzdem müssen die dann noch überarbeitet werden (besonders wenn z.B. Fußnoten enthalten sind) und milestones eingefügt werden.
Das beantwortet leider nicht meine Frage, ob ich alle und ob ich alle bis zum 11.10. machen soll/muss.
Wäre wichtig zwecks Zeiteinteilung und so.

**Comment by gerritbruening** [comment]
OK, wenn es geht, die Zeugnisse von oben bis einschl. Zeugnis-Nr. 41 (http://dev.faustedition.net/testimony/tille_310), weil man sie gleich am Anfang sieht, ergänzen. Oder zur Not die Hälfte davon, einfach damit wir nicht gleich am Anfang der Tabelle so viele Lücken haben (kosmetisches Kriterium; Zeitaufwand sehr pragmatisch handhaben; Fußnoten zur Not so kraus lassen wie sie sind).

**Comment by sandrakrause** [comment]
Die Zeugnisse sind jetzt alle eingefügt.
Kosmetische Behandlung auf ein Minimum beschränkt. Kann man ja bei Gelegenheit nach 1.0 noch aufhübschen, wo nötig.
Die milestones sind alle eingefügt.
Lediglich bei ticknor1876 musste ich die genaue Stelle raten, da ich den Biedermann nicht zur Hand hatte.
@gerritbruening könnten Sie mir vielleicht den Gefallen tun, die Stelle von bie3_4385 zu überprüfen?

**Comment by gerritbruening** [comment]
Habe Biedermann gerade auch nicht zur Hand. @dietmarpravida?

**Comment by gerritbruening** [comment]
> Die Zeugnisse sind jetzt alle eingefügt.

Großartig, danke!

**Comment by dietmarpravida** [comment]
4385 beginnt mit "Professor Riemer, who is..." und endet mit "...unsatisfied retirement"

**Comment by dietmarpravida** [comment]
#655 ist noch offen, ich mede mich morgen dazu

**Comment by sandrakrause** [comment]
Danke!
